.github/vacuum-ruleset.yaml

```diff
@@ -602,3 +602,14 @@ rules:
         then:
             function: typedEnum
         type: validation
+    require-property-descriptions:
+        category:
+            description: Documentation is really important, in OpenAPI, just about everything can and should have a description. This set of rules checks for absent descriptions, poor quality descriptions (copy/paste), or short descriptions.
+            id: descriptions
+            name: Descriptions
+        description: Schema properties must have a description.
+        given: $.components.schemas[*].properties[*]
+        then:
+            field: description
+            function: truthy
+        severity: error
```

Comment:
Yes, this is checking something different than each of those rules. The component rule checks only for a description on the schema object itself; the parameter rule checks only for descriptions on operation parameters but not the schemas they implement; the operation rule checks only the description of the operation itself.

I was surprised to not find a rule that did this already, but I couldn't see anything in [the docs](https://quobix.com/vacuum/rules/descriptions/).

Reply:
I did also take a look at [one of the existing rules](https://github.com/daveshanley/vacuum/blob/main/functions/openapi/operation_descriptions.go) to see if it might make sense to try to recreate it but pointed at schema properties instead, and it felt a little more complicated than I'm comfortable with, having zero golang experience. 

If you think this is worth pursuing though, let me know, and I can try to make it happen.

Reply:
OK, I see. Good to have your additional rule, no need to go for adding it any differently 👍